app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx

```diff
@@ -202,7 +202,15 @@ export const FormAudience = ({
             className="mx-5"
             controlId="totalEnrolledClients"
           >
-            <Form.Label>Expected number of clients</Form.Label>
+            <Form.Label>
+              Expected number of clients
+              {isMissingField("total_enrolled_clients") && (
+                <InlineErrorIcon
+                  name="clients"
+                  message="Expected number of clients must be a number greater than zero"
+                />
+              )}
+            </Form.Label>
```

Comment:
Looks like we were missing this icon? Seems like we shouldn't be, so I added it.

Reply:
Can't wait to be able to remove these altogether 🤞

Reply:
Yeah, I think an issue around this ended up in a "required for launch" epic in where we canceled a bunch of issues in favor of better functionality, let's leave it since the work was done but I think we'll remove it when we work on EXP-907